packages/@aws-cdk/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -285,6 +296,7 @@ export class Trail extends Resource {
       cloudWatchLogsRoleArn: logsRole?.roleArn,
       snsTopicName: this.topic?.topicName,
       eventSelectors: this.eventSelectors,
+      isOrganizationTrail: props.isOrganizationTrail ?? false,
```

Comment:
```suggestion
      isOrganizationTrail: props.isOrganizationTrail,
```
Let's just pass in what's provided. That way it doesn't impact templates unless it's explicitly provided.